hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/S3ATestUtils.java

```diff
@@ -1507,4 +1507,13 @@ public static void skipIfKmsKeyIdIsNotSet(Configuration configuration) {
     }
   }
 
+  /**
+   * Skip if a test doesn't use CSE.
+   */
+  public static void skipIfCSEIsNotEnabled(Configuration configuration) {
+    String encryption = configuration.get(Constants.SERVER_SIDE_ENCRYPTION_ALGORITHM, "");
+    if (!encryption.equals(S3AEncryptionMethods.CSE_KMS.getMethod())) {
```

Comment:
change this to skip if CSE is enabled.

Reply:
Sorry, I don't understand. Why would I change this method to skip if CSE is enabled? And the places where it's called (even though I added it you have the better context, so trying to understand it).

Reply:
The tests that require this are SSE tests which shouldn't skip if CSE-KMS is the encryption method used. It should've been skipped if CSE-KMS is enabled, to tell the user that SSE-KMS is not the method used.